src/Accordion/AccordionBody.tsx

```diff
@@ -1,35 +1,51 @@
-import React, { Component, ElementType } from 'react';
+import React, { ElementType, useCallback, useState } from 'react';
 import classNames from 'classnames';
 import { Transition } from 'react-transition-group';
 
-import { Util } from 'reactstrap';
+import { pick, omit } from '../utils';
 import type { TransitionProps } from 'react-transition-group/Transition';
 
-const {
-  // @ts-ignore
-  TransitionTimeouts,
-  // @ts-ignore
-  TransitionPropTypeKeys,
-  // @ts-ignore
-  TransitionStatuses,
-  // @ts-ignore
-  pick,
-  // @ts-ignore
-  omit
-} = Util;
-
 export type AccordionBodyProps = Partial<TransitionProps> & {
   tag?: ElementType;
   className?: string;
   active?: boolean;
   onToggle?: () => void;
 };
 
+// Taken from Reactstrap utils.js which are tken from bootstrap sass
+export const TransitionTimeouts = {
```

Comment:
Shouldn't we move these to some shared const files and then consume these? These appear to be generic